pkgs/development/python-modules/black/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonOlder, attrs, click }:
+
+buildPythonPackage rec {
+  pname = "black";
+  version = "18.4a0";
+  name = "${pname}-${version}";
```

Comment:
When using `buildPythonPackage` the `name` attribute defaults to `name = "${pname}-${version}";` so this line can be omitted.

Reply:
Good to know. Thanks!